### ISSUE TITLE
Court: Activate Changes to Notice of Receipt Of Petition (8652)

### ISSUE THREAD

**Issue by cholly75** [ustaxcourt/ef-cms#8003]
As a user, so that I may be see the changes to the notice of receipt of petition, I need the changes to be live in the production environment.

This is a story to track making the changes made in story 8652 active in the live DAWSON environment; they will be deployed but not active until the Court has authorized returning to in-person trial sessions.

***Update:** there is an additional change to the Notice that has been requested by the Court to the language in the section titled, "Electronic Access To Your Case:". We would like to make this change and activate the new template. Attached copy of PDF in the comments.

That section's language should read as follows:
"The Court encourages registration for DAWSON, the Court's electronic filing and case management system, so that you can electronically file and view documents in your case. For more information about electronic service, email dawson.support@ustaxcourt.gov, and a letter with eAccess instructions will be mailed to the address of record. If you do not register for eFiling, you must send the opposing party a copy of any document you file with the Court. To obtain further information about Tax Court proceedings, visit www.ustaxcourt.gov and select “Guidance for Petitioners.” "

## Pre-Conditions
8652 changes have been deployed but are not currently the active template for the Notice of Receipt of Petition

## Acceptance Criteria

- The updated change in this story and story 8652 are live and being used when generating for Notices of Receipt of Petition

- Currently existing Notices of Receipt of Petition are unaffected by this change and are not updated.

## IRS API Considerations
Do these changes impact the IRS API?

## Security Considerations
 - [ ] Does this work make you nervous about privacy or security?
 - [ ] Does this work make major changes to the system?
 - [ ] Does this work implement new authentication or security controls?
 - [ ] Does this work create new methods of authentication, modify existing security controls, or explicitly implement any security or privacy features?

## Notes
Companion to 8652

## Definition of Done (Updated 4-14-21)
**Product Owner**
 - [x]  Acceptance criteria have been met and validated on the Court's migration environment
 - [ ] Add scenario to testing document, if applicable (https://docs.google.com/spreadsheets/d/1FUHKC_YrT-PosaWD5gRVmsDzI1HS_U-8CyMIb-qX9EA/edit?usp=sharing)

**UX**
 - [ ] Business test scenarios to meet all acceptance criteria have been written
 - [ ] Usability has been validated
 - [ ] Wiki has been updated (if applicable) 
 - [ ] Story has been tested on a mobile device (for external users only)

**Engineering**
 - [x] Automated test scripts have been written
 - [x] Field level and page level validation errors (front-end and server-side) integrated and functioning
 - [x] Verify that language for docket record for internal users and external users is identical
 - [x] New screens have been added to pa11y scripts
 - [x] All new functionality verified to work with keyboard and macOS voiceover https://www.apple.com/voiceover/info/guide/_1124.html 
 - [x] READMEs, other appropriate docs, JSDocs and swagger/APIs fully updated
 - [x] UI should be touch optimized and responsive for external only (functions on supported mobile devices and optimized for screen sizes as required)
 - [x] Interactors should validate entities before calling persistence methods
 - [x] Code refactored for clarity and to remove any known technical debt
 - [x] Acceptance criteria for the story has been met
 - [x] Deployed to any Flexion environment for UX Review
 - [x] Deployed to the Court's migration environment

**Comment by cholly75** [comment]
[https://app.zenhub.com/files/152320868/b30fe89d-286c-488e-8a81-052012e2ad8b/download](https://app.zenhub.com/files/152320868/b30fe89d-286c-488e-8a81-052012e2ad8b/download) - PDF with desired text

**Comment by shumway-tm** [comment]
Updated PDF: 
![27F574AD-2240-45A9-AA73-2B4BD7772F94_1_105_c.jpeg](https://images.zenhubusercontent.com/618021eb47ddde1a5ec9cbe6/e0e275cc-9df6-43d0-b4ed-9a303eff6dcc)

**Comment by mrinsin** [comment]
Edited: Looks good to go!

**Comment by cholly75** [comment]
@TomElliottFlexion @shumway-tm - some cases the length of the new text results in the text of the notice appearing in the space of the envelope window where there should be only the petitioner name/address. I mocked up a PDF that changes the body text of the individual notice sections to 9 points down from 10.5 and shrunk the resulting subsections to eliminate whitespace. Confirmed with the Court that the printed and electronic result is appropriate. Can we make the same change here? Ideally, this would also preserve the Clerk's signature on the page as well instead of pushing it off to a page of its own.

- Change the body text of subsections "Your Docket Number", "Electronic Access To Your Case", "Change of Address", and "Request for Remote Proceedings" to 9pt. font
- Shrink the subsections so that no extra whitespace remains in or between them

[https://app.zenhub.com/files/152320868/74751447-1ee6-4438-8b6d-7babdd6edb85/download](https://app.zenhub.com/files/152320868/74751447-1ee6-4438-8b6d-7babdd6edb85/download)
